main/src/main/java/cgeo/geocaching/connector/al/ALApi.java

```diff
@@ -351,6 +353,7 @@ private static Geocache parseCache(final JsonNode response) {
             cache.setArchived(response.get("IsArchived").asBoolean());
             cache.setHidden(parseDate(response.get("PublishedUtc").asText()));
             // cache.setFound(parseCompletionStatus(response.get("CompletionStatus").asInt())); as soon as we're using active mode
+            cache.setFound(response.get("IsComplete").asBoolean());
```

Comment:
Maybe you could remove the outdated comment about found state above.

Reply:
Yes, actually wanted to but just forgot after all the excitment ;)

Reply:
done